src/backend/gl/src/command.rs

```diff
@@ -465,8 +484,24 @@ impl command::RawCommandBuffer<Backend> for RawCommandBuffer {
         self.push_cmd(Command::BindIndexBuffer(ibv.buffer.raw));
     }
 
-    fn bind_vertex_buffers(&mut self, _vbs: hal::pso::VertexBufferSet<Backend>) {
-        unimplemented!()
+    fn bind_vertex_buffers(&mut self, vbs: hal::pso::VertexBufferSet<Backend>) {
+        if vbs.0.len() == 0 {
```

Comment:
Let's assert here. Zero vertex buffers should be/is invalid.

Reply:
Is this a limitation of any of the backends? With GL at least I thought it was fine to avoid VBOs entirely and generate vertices using geometry shaders, or use `gl_VertexID` (i.e. full-screen triangle without a VBO), etc.

Reply:
@msiglreith 
> Zero vertex buffers should be/is invalid

where? why?

Reply:
It's a common pattern in Vulkan that no-op calls are invalid.
In particular for bind_vertex_buffers: https://www.khronos.org/registry/vulkan/specs/1.0/man/html/vkCmdBindVertexBuffers.html
```
bindingCount must be greater than 0
```

Reply:
> With GL at least I thought it was fine to avoid VBOs entirely

That's totally fine, but then you wouldn't call `bind_vertex_buffers` at all.

Reply:
@msiglreith I think we'll be fine with allowing that. The portability layer can assert if it has to, but gfx-hal doesn't have to do this.

Reply:
Makes sense. I thought it was possible to explicitly unbind vertex buffers but it appears that’s not allowed